robottelo/vm.py

```diff
@@ -272,6 +274,17 @@ def run(self, cmd):
 
         return ssh.command(cmd, hostname=self.ip_addr)
 
+    def fetch_domain(self):
+        """Fetches domian name from the domain of the provisioning
+        server as it's bridge based network.
+
+        """
+        domain = ssh.command(
+            u'hostname -d',
+            hostname=self.provisioning_server
+        )
+        return domain.stdout[0]
+
     def fetch_hostname(self):
         """Fetches short hostname from client and the domain from provisioning
```

Comment:
Plan is to remove this function and use vm.hostname , to avoid using `run` or `ssh.command`. Will remove this along with the test that uses this in my next PR.